pyproject.toml

```diff
@@ -48,6 +48,9 @@ pyodbc = { version = "~4.0.30", optional = true }
 sphinx-bootstrap-theme = "^0.8.0"
 Sphinx = "^4.2.0"
 tox = "^3.24.4"
+elasticsearch = "7.13.4"
```

Comment:
I wonder - can / should we use https://github.com/opensearch-project/opensearch-py instead? Released just 9 days ago, maintained by opensearch team... I don't know if there are any major differences in the API

Reply:
opensearch-py library is still in the works.
We'll use it once it will be production ready.

We're following the service team recommendation:
https://aws.amazon.com/blogs/opensource/keeping-clients-of-opensearch-and-elasticsearch-compatible-with-open-source/
https://opensearch.org/docs/clients/index/

Reply:
I've replaced elasticsearch-py with opensearch-py